features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/extensions/store/app/am-store-apis/apis/application.jag

```diff
@@ -18,7 +18,7 @@
  *
  */
 require('/modules/store.js').exec(function (ctx) {
-    var log = new Log();
+    var log = new Log('application_endpoint');
```

Comment:
Hi Chandana,

This is was discussed in one of my code reviews and this was renamed as per Udara's comment.

Thanks,